text/1071-cell.md

```diff
@@ -0,0 +1,296 @@
+---
+stage: accepted
+start-date: 2025-01-19T00:00:00.000Z
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1071
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+<-- Replace "RFC title" with the title of your RFC -->
+# Introduce `Cell` 
+
+## Summary
+
+This RFC introduces a new tracking primitive, which represents a single value, the `Cell`.
+
+## Motivation
+
+The `Cell` is part of the "spreadsheet analogy" when talking about reactivity -- it represents a single tracked value, and can be created without the use of a class, making it a primate candidate for demos[^demos] and for creating reactive values in function-based APIs, such as _helpers_, _modifiers_, or _resources_. They also provide a benefit in testing as well, since tests tend to want to work with some state, the `Cell` is wholly encapsulated, and can be quickly created with 0 ceremony. 
+
+This is not too dissimilar to the [Tracked Storage Primitive in RFC#669](https://github.com/emberjs/rfcs/blob/master/text/0669-tracked-storage-primitive.md). The `Cell` provides more ergonomic benefits as it doesn't require 3 imports to use. 
```

Comment:
Could you explain what the long-term intention with these kinda competing APIs is? Should we eventually deprecate the tracked-storage API, and tell users to only use `Cell`?

Reply:
I agree with point from @simonihmig, if we already have such low-level primitive, seems `Cell` could be build on top of it. Or, `Cell` could be hidden inside `tracked-storage-primitive` functions. Have both seems very strange, and deprecating `tracked-storage-primitive` seems strange too.

If we rewrite `tracked-storage-primitive` using Cell, there is no RFC needed (because it's part of private API).

Reply:
yea, I'll make a note